test/fixtures/poro.rb

```diff
@@ -62,12 +62,10 @@ module Spam; end
   attributes :id, :title, :body
 
   has_many :comments
-  belongs_to :blog
-  belongs_to :author
-
-  def blog
+  belongs_to :blog do
     Blog.new(id: 999, name: 'Custom blog')
```

Comment:
n/m it just looks weird to create an artificial association, but that's pre-existing